Editors/SoundEditor.cpp

```diff
@@ -114,7 +121,8 @@ void SoundEditor::on_loadButton_clicked() {
         // QString newData = GetModelData(Sound::kDataFieldNumber).toString();
         // TODO: Copy data into our egm and reset the path
         // SetModelData(Sound::kDataFieldNumber, lastData);
-      } else qDebug() << "Failed to load gmx sound";
+      } else
+        qDebug() << "Failed to load gmx sound";

```

Comment:
Did clang-format do this? It should have put {} around this else clause.

Reply:
it did but i dont think clang format puts brackets